### PR TITLE
Add android app page

### PR DESCRIPTION
**Pull request by apt105**

### CODE REVIEW

File: pages/getting-started/setup-client/android.md [hunk 1]
```diff
@@ -7,3 +7,20 @@ parent: Setup Client
 grand_parent: Getting Started
 ---
 # Android Instructions
+
+Our Android application is available to download from Google Play
+
+[![Download at https://play.google.com/store/apps/details?id=com.geeksville.mesh](https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png)](https://play.google.com/store/apps/details?id=com.geeksville.mesh&referrer=utm_source%3Dgithub-homepage)
```
Comment (jfirwin) [inline pages/getting-started/setup-client/android.md:13]: I might swap this out for a smaller icon, or add some CSS scaling in the future.

**Review by jfirwin — APPROVED** [review]
Everything looks good. I have some changes on my local branch that might move these files around a bit, but I'm happy with it! Thanks so much for your contributions. I've been pretty focused on the overall layout, and haven't had time to dig into content yet, so this is excellent.

**Comment by jfirwin** [comment]
Looks like those images aren't quite right. The logo is handled in the config.yml, but it looks like it references the whole url "https://jfirwin.com/meshtastic-documentation/assets/meshtastic-logo.png" I'll try fixing that, weird behavior on this.

File: pages/getting-started/setup-client/android.md (hunk 1 above)
Comment (apt105) [reply pages/getting-started/setup-client/android.md:13]: Aye, I wasn't too sure how to change it's size in markdown

**Comment by apt105** [comment]
I did wonder about the possibility of absolute vs relative links, but given the yml thought it would probably work, but alas. I'm happy to keep throwing more stuff on as I can, but feel free to move/change anything.

**Pull request merged**